website/content/api-docs/auth/cert.mdx

```diff
@@ -257,6 +257,7 @@ Sets a named CRL.
 
 - `name` `(string: <required>)` - The name of the CRL.
 - `crl` `(string: <required>)` - The PEM format CRL.
+- `url` `(string: <required>)` - The URL of a CRL distribution point.
```

Comment:
Technically we need to specify either `crl` or `url` not both. Looking at the code atm we prefer `crl` over `url` if both are specified. 

@schavis do we have a preferred style on how to specify parameter requirements when we require one or the other but not both?

```suggestion
- `crl` `(string: "")` - The PEM format CRL.
- `url` `(string: "")` - The URL of a CRL distribution point.

**Note**: Either 'crl' or 'url' parameters must be provided, if both are provided, 'crl' is used.
```